auth/auth/auth.py

```diff
@@ -717,7 +717,7 @@ async def get_session_id(request):
     return session.get('session_id')
 
 
-@routes.get('/api/v1alpha/verify_dev_credentials')
+@routes.route('*', '/api/v1alpha/verify_dev_credentials')
```

Comment:
This change is because Envoy's built-in auth request mechanism sends the whole incoming request as is to the "auth cluster". So if I'm sending a POST to batch.hail.is, it will also send a POST to authenticate that request. The easiest way I could find to get around this was to register this authentication endpoint under all HTTP methods.